test/pycheck/motherduck_test.py

```diff
@@ -151,6 +153,49 @@ def test_md_multiple_databases(pg_two_dbs, md_test_user):
         assert cur2.sql(list_tables_query) == ("public", "user2_t1")
 
 
+@pytest.mark.timeout(0)
+@pytest.mark.skipif(not can_run_md_multi_user_tests(), reason="needs multiple users")
+def test_md_read_scaling_two_connections(pg_two_dbs, md_test_user):
+    cur1, cur2 = pg_two_dbs
+    ## FIXME - If the DB is not unique, the test fails
+    db_name = "test_md_db" + str(uuid.uuid4()).replace("-", "")
+    user1_spec = {"database": db_name, "token": md_test_user["token"], "hint": "cli1"}
+    user2_spec = {
+        "database": db_name,
+        "token": create_rs_token(md_test_user)["token"],
+        "reset_db": False,
+        "hint": "cli2",
+    }
+
+    with MDClient.create(user1_spec) as cli1:
+        # Create a table in user1's DB...
+        cli1.run_query("DROP TABLE IF EXISTS t1;")
+        cli1.run_query("CREATE TABLE t1(a int, b varchar)")
+        cli1.run_query("INSERT INTO t1 VALUES (41, 'hello world')")
+
+        with MDClient.create(user2_spec) as cli2:
+            # First check in the second connection that we can see the table
+            for _ in wait_until("Failed to get t1 records", timeout=60):
```

Comment:
```suggestion
            for _ in wait_until("Failed to get t1 records", timeout=70):
```
Let's make it 70 for a bit of extra leeway.